packages/shared/src/components/widgets/PostUsersHighlights.tsx

```diff
@@ -172,19 +173,16 @@ export const UserHighlight = (props: UserHighlightProps): ReactElement => {
       >
         <div
           className={classNames(
-            'ml-4 flex min-w-0 flex-1 flex-col',
+            'flex min-w-0 flex-1 flex-col',
             className?.textWrapper,
           )}
         >
           <div className="flex">
             <ProfileLink
-              className={classNames(
-                'truncate font-bold typo-callout',
-                className?.name,
-              )}
+              className={classNames('font-bold typo-callout', className?.name)}
               href={permalink}
             >
-              {name}
+              <TruncateText>{name}</TruncateText>
```

Comment:
Please just double check manually this truncates correctly, because of lot of wrappers this component is known to break and clip in a weird ways. 🙏

Reply:
It works! 
<img width="401" alt="image" src="https://github.com/user-attachments/assets/eb6a8f7a-d233-4e56-9c5d-593449fe6a6f">